specs/agents/tracing-sampling.md

```diff
@@ -63,11 +63,13 @@ Metrics will be more accurate when the sampling rate is high.
 With lower sampling rates the server is able to calculate representative, but less accurate, metrics.
 If the sampling rate is 0 then no metrics will be calculated at all.
 
-Agents will record the sampling rate on transactions and spans as `sample_rate`. e.g.
+When the sampling rate is available Agents _must_ record the sampling rate on transactions and spans as `sample_rate`, e.g.
```

Comment:
```suggestion
When the sampling rate is available Agents MUST record the sampling rate on transactions and spans as `sample_rate`, e.g.
```

All caps is the pattern that's been established in other specs.

Reply:
Is it worth updating the rest of this sub-doc to confirm to this convention? Ideally we should probably do it across all the agents spec sub-docs but I think that will be too much for this PR.

Reply:
My bad, you were following the existing doc's patterns. I would update this doc, yes, but leave the rest for a separate PR.

Reply:
Done - please take a look.